libs/executors/include/hpx/executors/limiting_executor.hpp

```diff
@@ -29,166 +28,330 @@
 //
 // --------------------------------------------------------------------
 namespace hpx { namespace parallel { namespace execution {
-    template <typename Executor>
-    struct limiting_executor
-    {
-        // --------------------------------------------------------------------
-        // For C++11 compatibility
-        template <bool B, typename T = void>
-        using enable_if_t = typename std::enable_if<B, T>::type;
-
-        // --------------------------------------------------------------------
-        //
-        // --------------------------------------------------------------------
-        limiting_executor(std::size_t lower, std::size_t upper,
-            bool block_on_destruction = true)
-          : executor_(Executor())
-          , count_(0)
-          , lower_threshold_(lower)
-          , upper_threshold_(upper)
-          , block_(block_on_destruction)
-        {
-        }
-
-        limiting_executor(const Executor& ex, std::size_t lower,
-            std::size_t upper, bool block_on_destruction = true)
-          : executor_(ex)
-          , count_(0)
-          , lower_threshold_(lower)
-          , upper_threshold_(upper)
-          , block_(block_on_destruction)
-        {
-        }
+    namespace experimental {
```

Comment:
I had `hpx::execution::experimental` in mind (without `parallel`) since the execution functionality is slowly moving into `hpx::execution`. The namespace will anyway change once the limiting executor becomes non-experimental, so I don't mind if you prefer to keep this namespace. It would be slightly nicer to drop one namespace though.

Reply:
done

Reply:
Thanks!